airflow/providers/common/sql/provider.yaml

```diff
@@ -25,6 +25,7 @@ state: ready
 source-date-epoch: 1709555283
 # note that those versions are maintained by release manager - do not update them manually
 versions:
+  - 1.12.0
```

Comment:
This is also not needed.
Release manager knows to set the version of bug fix / feature upon release

Reply:
See the other comment @eladkal . We MUST add version because otherwise the build will not work as we are doing >= 1.12.0 in the other operators.

Reply:
BTW. We've always been doing it when we added new features in common.sql that other providers depended on.

Reply:
ah right!